src/store/actions/voting.js

```diff
@@ -117,10 +117,12 @@ export const votesRetrieved = () =>
   async (dispatch, getState) => {
     const { account, network } = getState();
     const address = account.info[tokenMap.LSK.key].summary.address;
-    const votes = await getVotes({ network, params: { address } });
-
-    dispatch({
-      type: actionTypes.votesRetrieved,
-      data: votes.data,
-    });
+    try {
+      const votes = await getVotes({ network, params: { address } });
+      dispatch({
+        type: actionTypes.votesRetrieved,
+        data: votes.data,
+      });
+      return 0;
+    } catch (exp) { return 0; }
```

Comment:
The action should always dispatch. Although the action type and data might be different according to the API response, there should always be an action dispatched. This brings consistency to the behavior of the action function.